plugins/armory-agent/patch-clouddriver-docker.yaml

```diff
@@ -12,12 +12,12 @@ spec:
     profiles:
       clouddriver:
         kubernetes:
-          enabled: true    # This is not needed if spinnaker already has kubernetes V2 accounts enabled by other files
+          enabled: true # This is not needed if Spinnaker already has Kubernetes V2 accounts enabled by other files
 
         sql:
-          enabled: true    # kubesvc depends on clouddriver using SQL. See patch-sql-clouddriver for full configuration
+          enabled: true # kubesvc depends on Clouddriver using SQL. See patch-sql-clouddriver for full configuration
         redis:
-          enabled: true    # kubesvc depends on redis for internal pub/sub communication
+          enabled: true # kubesvc depends on Redis for internal pub/sub communication
```

Comment:
And ln23 as well. I believe redis usage with scale agent is deprecated. I believe we can leave out `redis.enabled`  and change `kubesvc.cluster` to `kubernetes` instead of `redis` @AdanUrbanReyesArmory can confirm

Reply:
Thats right redis was deprecate since these agent plugin versions :  [v0.11.1](https://github.com/armory-io/agent-k8s-spinplug/releases/tag/v0.11.1), [v0.10.40](https://github.com/armory-io/agent-k8s-spinplug/releases/tag/v0.10.40), [v0.9.56](https://github.com/armory-io/agent-k8s-spinplug/releases/tag/v0.9.56), [v0.8.64](https://github.com/armory-io/agent-k8s-spinplug/releases/tag/v0.8.64)

Reply:
I updated the docker file from what's in the docs, which I'm guessing is more recent